build_tools/mako/utils.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```

Comment:
Can we give this file a more creative name? It's not really utils. More like configuration. Even `configuration.py` would be better IMO